application/common/components/adapters/SagePeopleIdStore.php

```diff
@@ -22,12 +22,15 @@ class SagePeopleIdStore extends IdStoreBase
     const PROP_LOCKED = 'fHCM2__User__r.jaars_Locked_From_IdP__c';
     const PROP_REQUIRE_MFA = 'fHCM2__User__r.jaars_Require_2sv_with_IdP__c';
 
+    const DEFAULT_QUERY_CONDITIONS = "fHCM2__Employment_Status__c='Active' AND fHCM2__User__c!=null AND fHCM2__Department__c!='a0H1U000001NKk4UAG'";
```

Comment:
This looks like hard-coded data, especially the "a0H1..." bit. Should that be dynamic?

Reply:
Yes, I decided to keep it this way for now, but added a Github issue to remove it.